src/map/battle.c

```diff
@@ -4855,15 +4860,15 @@ static struct Damage battle_calc_weapon_attack(struct block_list *src, struct bl
 
 	if (sd && !skill_id) {
 		//Check for double attack.
-		if (( (skill_lv=pc->checkskill(sd,TF_DOUBLE)) > 0 && sd->weapontype1 == W_DAGGER )
+		if (( (skill_lv = pc->checkskill(sd,TF_DOUBLE)) > 0 && sd->weapontype1 == W_DAGGER )
```

Comment:
since you fix this, complete it

Reply:
Do you mean the spaces around parenthesis? Please elaborate.

Reply:
that and `sd, TF_DOUBLE`